code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -905,13 +907,13 @@ datum
 
 			on_add()
 				if (istype(holder) && istype(holder.my_atom) && hascall(holder.my_atom,"add_stam_mod_max"))
-					remove_buff = holder.my_atom:add_stam_mod_max("consumable_bad", -20)
+					remove_buff = holder.my_atom:add_stam_mod_max("r_pancuronium", -30)
```

Comment:
Looks like you changed this from `-20` to `-30`.

Reply:
Yeah, that was intentional. 20 seemed too low for panc